### DISCUSSION TITLE
Custom kind text and icon

### DISCUSSION THREAD

**Discussion by stefanboca** [saghen/blink.cmp#435] [Q&A]
Is there any plan to add a way to easily override kind text and icons? I see something similar was mentioned [here](https://github.com/Saghen/blink.cmp/pull/253#issuecomment-2454984622). nvim-cmp for instance adds a [`cmpCompletionExtension`](https://github.com/hrsh7th/nvim-cmp/blob/ca4d3330d386e76967e53b85953c170658255ecb/lua/cmp/types/lsp.lua#L284) field to its completion items, which contains fields for [`kind_text` and `kind_hl_group`](https://github.com/hrsh7th/nvim-cmp/blob/ca4d3330d386e76967e53b85953c170658255ecb/lua/cmp/types/lsp.lua#L261).

**Comment by voidstar-null** [comment]
You may want to look at this:

#458
